test/classes/bradc/bitOpOnClass.chpl

```diff
@@ -1,29 +1,32 @@
 class MyClass {
-var len: int;
-var dom: domain(1);
-var arr: [dom] int;
-proc MyClass(length: int) {
-  this.len = length;
+  var len: int;
+  var dom: domain(1);
+  var arr: [dom] int;
+
+  proc init(length: int) {
+    this.len = length;
     this.dom = {1..length};
-      forall (a,i) in zip(this.arr, 13..) do
-        a = i;
-        }
+    super.init();
```

Comment:
This looks like another case where the refined "where does the compiler insert its default field inits
 rule we discussed (switching to "just after the previous field init" rather than "just before the next field init") would permit all the code here to be on the same side of the super.init().  The more I think about this rule, the more I think we should implement it sooner to see if it leads to any surprises and to be able to make use of it in cases like this.